docker/README.md

```diff
@@ -1,14 +1,15 @@
 ### Building Redox using Docker images with the toolchain
 
-*All you need is git, make, qemu, fuse and docker. The method requires a non-privileged user able to run the `docker` command, which is usually achieved by adding the user to the `docker` group.*
+*All you need is rustup, git, make, qemu, fuse and docker. The method requires a non-privileged user able to run the `docker` command, which is usually achieved by adding the user to the `docker` group.*
```

Comment:
I'm not sure why syncing submodules from within a container would be significantly slower than doing it from a host system. I've tested it and it took ~10 minutes to do `make pull` on a fresh repo from within a container, approximately the same time it takes on the host system. Unfortunately,  I've run into some github authentification problems while doing this because I was using my own fork. So, in the end, it looks like syncing submodules from the host system is indeed the best way for now.

> I ended up with xargo not being found any more. 

I guess you didn't use quotes, so your shell thought `make all` was a separate command and tried to run it on your host machine directly.